public/index.html

```diff
@@ -238,6 +235,22 @@ <h3>Add New Schedule</h3>
                 </div>
                 <button type="submit" class="submit-btn">Add Schedule</button>
             </form>
+            <script>
+                document.addEventListener("DOMContentLoaded", () => {
```

Comment:
There's already a declaration for this event in line 82. Cn you use that instead?

Reply:
Ok you mean the event listener too, wait